unity/Assets/Scripts/AgentManager.cs

```diff
@@ -643,41 +651,20 @@ private void addImageSynthesisImageForm(WWWForm form, ImageSynthesis synth, bool
 		if (shouldRender) {
 			// we should only read the screen buffer after rendering is complete
 			yield return new WaitForEndOfFrame();
-			if (synchronousHttp) {
-				// must wait an additional frame when in synchronous mode otherwise the frame lags
-				yield return new WaitForEndOfFrame();
-			}
+			yield return new WaitForEndOfFrame();
```

Comment:
Can you add the comment back to have reference of why we are waiting twice?

Reply:
Done